spec/system/admin/products_v3/products_spec.rb

```diff
@@ -99,6 +103,20 @@
         expect_products_count_to_be 1
       end
     end
+
+    context "search by category" do
+      it "has a category select" do
+        expect(page).to have_selector "select#category_id"
+      end
+
+      it "can search for a product" do
+        search_by_category "Category 1"
+
+        expect(page).to have_select "category_id", selected: "Category 1"
+        expect_page_to_be 1
+        expect_products_count_to_be 1
```

Comment:
We could also check that the correct product is showing (we'll need to give the product a name).

Reply:
Yes!